scripts/jenkins/ed.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Kops requires user input through an editor. This shell script simply returns
+# the file input '$1' to STDOUT which enables us to script editing nodes through
```

Comment:
I'm not sure this is 100% correct. Maybe @alexsomesan can comment. IIRC how this hack works is that kops calls `$EDITOR current-state-of-file after-edit-done-file`. And as we can pass the first one directly to kops and we are sure this is what we want as it has been checked in to git we can just write it to the "destination" file.

Either way naming $1 and $2 would definitely help.